client/src/components/game/containers/PlayedCards.jsx

```diff
@@ -0,0 +1,17 @@
+import './playedCards.scss';
+
+import React from 'react';
+
+import { PlayedCard } from '../cards/PlayedCard';
+
+const PlayedCards = ({ numberOfCards }) => {
+  let playedCards = Array(numberOfCards);
+  for (let i = 0; i < numberOfCards; i++) {
+    let card = <PlayedCard></PlayedCard>;
```

Comment:
According to React docs the key prop is by default the element's index. Should I simply add the `key=i` prop in `<PlayedCard>` (Then the behaviour doesn't change, but the error is suppressed), or is there a better way to add the key?

Reply:
You're right, using index as a key like `<PlayedCard key={i}/>` is anti-pattern in many cases and used here would only suppress warning. Imo after client-server implementation it will be easier and more intuitive to find proper real unique key for each card